### DISCUSSION TITLE
Have to restart Webpack (watch mode) if there's an error in tailwind.config.js

### DISCUSSION THREAD

**Discussion by emilmuller** [tailwindlabs/tailwindcss#10927] [Help]
I'm not sure if this is an issue with tailwind, or postcss, or webpack.

When running webpack in watch mode, and if you (while webpack is running) edit tailwind.config.js to have an error in it (e.g. syntax error), things will crash. Even if you fix the error in tailwind.config.js, you will have to restart webpack to make it work again.

This can be somewhat frustrating if you have auto save enabled in vscode. As you are typing, you'll obviously end up with syntax errors until you have finished typing.

Kind regards, Emil

**Comment by thecrypticace** [comment]
Can you provide a reproduction? I don't think this is a Tailwind CSS issue (but not 100% sure).
